JavaScript/JavaScriptSDK/SendBuffer.ts

```diff
@@ -183,6 +187,16 @@ module Microsoft.ApplicationInsights {
             var sentElements = this.getBuffer(SessionStorageSendBuffer.SENT_BUFFER_KEY);
             sentElements = sentElements.concat(payload);
 
+            if (sentElements.length > SessionStorageSendBuffer.MAX_BUFFER_SIZE) {
```

Comment:
I understand not accepting new items to be sent when over quota but something feels wrong about losing track of data that really was already sent, what am I missing?